bases/br_ms_sim/code/sub/dbc2csv.R

```diff
@@ -0,0 +1,24 @@
+
+#---------------------------------------------------------#
+# preface
+#---------------------------------------------------------#
+
+library(foreign)
+library(read.dbc)
+
+setwd("~/Dropbox/Academic/Data/Brazil/DataSUS/Sistema de Informacoes sobre Mortalidade (SIM)")
+
+#---------------------------------------------------------#
+# build 
+#---------------------------------------------------------#
+
+file.names <- grep("DO", dir("input/dbc", pattern=".dbc"), value=TRUE)
+
+dir.create("input/csv")
+
+for(file in file.names){
+  
+  df.file <- read.dbc(paste("input/dbc/",file,sep=""))
```

Comment:
Ricardo, aqui talvez seja bom mudar para paste0 ou para glue, eles são mais seguros né e para caso alguém que for reproduzir entenda que todas as partes tem que estar grudadas uma na outra.

Reply:
Boa já mudei lá.